doc/release/new-policy.rst

```diff
@@ -0,0 +1,159 @@
+
+Changes
+-------
+
+There are several significant changes from the legacy release policy:
+
+*   The third number in the version name doesn't distinguish between
+    alpha, beta and release version anymore.
+
+*   In the legacy release policy, 1.10 was a long-term support (LTS) series,
+    while 2.x.y had "stable releases", but wasn't an LTS series.
+
+    Now both series are long-term supported.
+    The intended support time is at least two years.
+
+A release series lifecycle
+--------------------------
+
+A release series goes over the following stages:
+
+..  container:: table
+
+    ..  rst-class:: left-align-column-1
+    ..  rst-class:: left-align-column-2
+
+    ..  list-table::
+
+        *   -   **Stage**
+            -   **Versions to publish**
+            -   **Example**
+
+        *   -   Early development
+            -   Alpha, beta, release candidate
+            -   3
+
+        *   -   Support
+            -   Release candidate, release
+
+        *   -   End of life
+            -   N/A
+
+
+The sections below describe those stages in detail.
+
+A release series stages
+-----------------------
+
+Early development
+~~~~~~~~~~~~~~~~~
+
+The stage goes on until a first release. Alpha, beta versions and pre-releases
+are published within this stage.
+
+In fact, this stage splits into two phases: development of a new functionality
+and its stabilization.
+
+A premature functionality might be removed on the alpha/beta stage, but it will
+not be removed after the publication of a release candidate.
+
+Support
+~~~~~~~
+
+The stage starts when a first release is published. The release series now is
+an object of only backward compatible changes.
+
+At this stage, we're fixing all known security problems and fixing all found
+degradations since the previous series.
+
+A series receives degradation fixes and other bugfixes till the
+end of life.
+
+The decision of whether to fix a particular problem in a particular release series
+is based on the impact of the problem, risks around backward compatibility and the
+complexity of backporting a fix.
+
+A release series might receive new features at this stage, but only in a
+backward compatible manner. A release candidate might be published for a new
+functionality before a release.
+
+During the support period we're adding new versions of supported Linux distros
+to our build infrastructure.
+
+A support period might be extended.
+
+See the **Delivery channels** section for information about a particular bug in the release
+or about the resolve of a feature request.
+
+End of life
+~~~~~~~~~~~
+
+A series reaches the end of life (EOL) when the last release in the series is
+published. The series will not receive updates anymore.
+
+In modules, connectors and tools, we don't guarantee support of a release series
+that reaches EOL.
+
+A release series cannot reach EOL until the vast majority of productions
+(where we have commitments / SLA) will be updated to a newer series.
+
```

Comment:
What I really like from the 'Version string meaning' section of the [RFC](https://github.com/tarantool/tarantool/discussions/6182): paragraphs, which clarify how pre-releases are intended to be used:

> ### Nightly build
>
> Those versions are not supposed to be used by customers. A versions string
contains `-dev` postfix.
>
> ### Alpha
>
> An alpha version is for early adopters and developers of dependent components
(such as connectors and modules).
>
> <...>
>
> ### Beta
>
> A beta version is good to start developing of a new application.
>
> <...>
>
> ### Release candidate
>
> A release candidate fits good to setup a staging server.
>
> <...>
>
> ### Release
>
> A release is the version that is ready for production usage.
>
> <...>

Other details are also interesting, but given paragraphs look as 'must have' for the release policy description.